src/hera.cpp

```diff
@@ -42,6 +42,10 @@ using namespace std;
 using namespace wasm;
 using namespace HeraVM;
 
+struct hera_instance : evm_instance {
+  bool fallback = false;
```

Comment:
Does this mean that `fallback` is a member **after** the members of `evm_instance`?

Reply:
Yes.